backend/inspirehep/migrator/tasks.py

```diff
@@ -439,6 +439,10 @@ def run_orcid_push(uuids):
 
 @shared_task(ignore_results=False, queue="migrator", acks_late=True)
 def run_hal_push(uuids):
+    if not current_app.config["FEATURE_FLAG_ENABLE_HAL_PUSH"]:
```

Comment:
I'm not sure if this is the place where we want to do this check. As It will already create a task (unnecessary). It should be IMHO checked before we call this task.